pkg/endpoint/policy.go

```diff
@@ -498,6 +499,230 @@ func (e *Endpoint) IngressOrEgressIsEnforced() bool {
 		e.Opts.IsEnabled(OptionEgressPolicy)
 }
 
+func (e *Endpoint) GetPortNumber(port string) (uint16, error) {
+	i, err := strconv.Atoi(port)
+	if err != nil {
+		// FIXME check for ports names here if it doesn't exist, the throw initial error
+		return 0, err
+	}
+	return uint16(i), nil
+}
+
+func (e *Endpoint) regeneratePolicyNew(owner Owner, opts models.ConfigurationMap) (*policy.Policy, *policy.Policy, uint64) {
```

Comment:
Do you intend for this function to handle `PolicyEnforcementMode` or will it be done by the part that actually pushes the generated policies into the maps?

Reply:
It can be done yes.